src/main/java/org/dita/dost/reader/DitaValReader.java

```diff
@@ -147,6 +149,10 @@ public void startElement(final String uri, final String localName, final String
             if (f.isAbsolute()) {
                 imageList.add(f);
                 relFlagImageList.add(getRelativePath(ditaVal, f));
+            } else if (atts.getValue(DITA_OT_NS, ATTRIBUTE_NAME_IMAGEREF_URI) != null) {
+                final URI imageuri = toURI(atts.getValue(DITA_OT_NS, ATTRIBUTE_NAME_IMAGEREF_URI));
```

Comment:
`toURI` is meant for cases where the input may be invalid URI or system path. In cases where you know the value is a valid URI, you should just use `URI.create(String)` method. If you know the value is a URI but not sure if it's valid, then use `new URI(String)` and catch exception.